library/include/hipblas.h

```diff
@@ -12589,6 +12589,29 @@ HIPBLAS_EXPORT hipblasStatus_t hipblasDspr2_64(hipblasHandle_t   handle,
                                                const double*     y,
                                                int64_t           incy,
                                                double*           AP);
+
+// 64-bit interface
+HIPBLAS_EXPORT hipblasStatus_t hipblasSspr2Batched_64(hipblasHandle_t    handle,
```

Comment:
I think this (and the strieded_batched versions) were moved around accidentally, this should be around L12709, for example. I think this is what's causing the CI documentation failures.

Reply:
Thanks for catching, something went wrong with the rebase.